### DISCUSSION TITLE
WIP triage labels

### DISCUSSION THREAD

**Discussion by duncdrum** [eXist-db/exist#4874] [Team Posts]
sneakpeek of the new labels, feedback welcome [gDocs](https://docs.google.com/drawings/d/1eVZAPShDy0ALiq3mhgzwcqndguekcfPpFjxJeoKS7OA/edit?usp=sharing)

**Comment by joewiz** [comment]
Wow, these look great, Duncan! Just a couple of observations:

- typo: xml-rdc -> xml-rpc
- what's "xquery-map"?
- what's "Build and Repo"?
- would parser-related issues go into "xquery"?

**Comment by duncdrum** [comment]
xquery-map is for map syntax related bugs, there is a lot of them
build and repo is for all things related to the build file and github repo there. this collapses a few old labels with only a few issues each
yes parser issues are xquery issues the difference between parser, lexer and engine aren't very easy to discern during triage

**Reply by adamretter** [reply]
May I suggest xdm-map instead, as Map is defined in the XDM spec and not in XQ spec.

**Comment by joewiz** [comment]
Great, thanks. That all makes sense.
